### PR TITLE
Done

### PR DESCRIPTION
**Pull request by bhollan**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-students.learn.co
```
Comment (adamjonas) [inline CNAME]: hi @bhollan. Please re-add this file.

**Comment by adamjonas** [comment]
The rest looks good. I'll merge when that CNAME is back. Thanks!

**Comment by bhollan** [comment]
Done.

Is it intentional that the instructions say to remove it and then commit that change?
From the [README.md](https://github.com/learn-co-students/learn-deploy-on-day-one-v-000/blob/master/README.md).
```
1. Delete the CNAME file and commit that change. Reminder: you can delete a file with rm CNAME. Make sure you git add ., then git commit -m "removing CNAME", then git push.
2. Create a new branch called gh-pages. Reminder: you can do this with git co -b gh-pages

```
Wouldn't it be better to have the branch created BEFORE the delete/commit?
I understand if it's intentional.  To make sure people can follow those instructions, to make sure the CNAME doesn't end up on their gh-pages branch, to make sure people know how to add simple files back, maybe even inspire some curiosity to look up what a CNAME file does.  But I just wanted to ask if it was intentional or accidental.  I followed the instructions.  :)

**Comment by adamjonas** [comment]
Good catch. I updated the readme. You shouldn't need to mess around with deployment for now... You did all the right things.

File: students/brian-holland.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  <body>
+    <div class="main-wrapper profile">
+      <header class="header">
+        <a href="http://learnpythonthehardway.org/">
+          <img class="learn-logo" src="../assets/img/logo.png" alt="Learn logo"/>
+        </a>
+      </header>
+      <div class="profile-banner" id="brian-holland-cover"></div>
+      <div class="vitals-container">
+        <div class="profile-photo" id="brian-holland-card"></div>
```
Comment (adamjonas) [inline students/brian-holland.html]: you are winning the cute kid contest

**Pull request merged**